lib/createsend/journey.rb

```diff
@@ -0,0 +1,62 @@
+module CreateSend
+  # Represents a journey and provides associated functionality
+  class Journey < CreateSend
+    attr_reader :journey_id
+
+    def initialize(auth, journey_id)
+      @journey_id = journey_id
+      super
+    end
+
+    # Get a full summary of a journey
+    def summary
+      response = get "/journeys/#{@journey_id}.json"
+      Hashie::Mash.new(response)
+    end
+
+    # Gets a list of all recipients of a particular email within a journey
+    def email_recipients(email_id="")
+      raise ArgumentError, 'Email must be a string' unless not email_id.nil? and email_id.kind_of? String
+
+      response = get_action email_id, "recipients"
```

Comment:
the journey email recipients can also be paged by date as well. This is different with campaign recipients because journey is working for a period of time vs just once off for campaign